src/Pages/List/Components/Refund.js

```diff
@@ -2,32 +2,43 @@ import React, { useState } from 'react';
 import styled from 'styled-components';
 import Tooltip from './Tooltip';
 
-const Refund = ({ toggleTooltip, value, selectedTooltip, sendToServer }) => {
-  const [refundable, setRefundalbe] = useState(false);
+const Refund = ({
+  value,
+  selectedTooltip,
+  setSeletedTooltip,
+  filteredCondition,
+  setFilteredCondition,
+}) => {
+  const [isRefundable, setIsRefundalbe] = useState(false);
 
   const moveSwitch = () => {
-    setRefundalbe(!refundable);
+    setIsRefundalbe(!isRefundable);

```

Comment:
useState의 return 값에 두번째 인자로 받는 함수
여기서는 setIsRefundalbe(적다보니 오타있네요 수정해주세요!)은
인자로 두가지 타입을 받을 수 있습니다!
첫번째는 우리가 흔히 사용하듯이 새로운 state로 대체될 값 `!isRefundable` 을 받으면 그 값으로 state를 대체하고
인자로 함수를 넣으면은 그 함수의 첫번째 인자에 prevState값(현재 state값)을 전달해줍니다 그리고 함수의 리턴값으로 새로운 state를 대체합니다!
그래서 만약 이전 state값에 의존해서 새로운 state를 만들어야 하는 경우
지금처럼 기존의 isRefundable값을 이용해서 새로운 state를 만들어야하는경우 함수형태의 인자를 전달하는것도 좋아보입니다!
```suggestion
    setIsRefundalbe(prevState => !prevState);
```